sdk/eventhub/Azure.Messaging.EventHubs/stress/stress-test-resources.json

```diff
@@ -5,7 +5,7 @@
     "_generator": {
       "name": "bicep",
       "version": "0.4.1318.3566",
-      "templateHash": "9849786101028970025"
```

Comment:
I think because you previously checked this in, the .gitignore won't automatically remove it. So you have to manually remove it with `git rm` and commit the file deletion.